indico/modules/events/surveys/templates/submission.html

```diff
@@ -0,0 +1,56 @@
+{% extends 'layout/base.html' %}
+
+{% block title %}
+    {% trans %}Survey{% endtrans %}
+{% endblock %}
+
+{% block subtitle %}
+    {% trans %}Submission{% endtrans %}
+{% endblock %}
+
+{% block content %}
+    <div class="page-content-description">
+        <p>
+            {% if submission.user %}
+                {% trans submitter_name=submission.user.full_name -%}
+                    Submitter: {{ submitter_name }}
+                {%- endtrans %}
+            {% else %}
+                {% trans -%}
+                    Submitter: <em>Anonymous</em>
+                {%- endtrans %}
+            {% endif %}
+        </p>
+        <p>
```

Comment:
i think using two paragraphs for this is  wrong from a semantic point of view (and the spacing between them looks weird, too). what about using a single paragraph and using a normal `<br>` to separate the two lines? that way you can even use the description block (which already adds the `<p>`) instead of putting it manually